network/p2p/middleware.go

```diff
@@ -159,7 +161,7 @@ func DefaultValidators(log zerolog.Logger, flowID flow.Identifier) []network.Mes
 	}
 }
 
-func (m *Middleware) topologyPeers() (peer.IDSlice, error) {
+func (m *Middleware) TopologyPeers() (peer.IDSlice, error) {
```

Comment:
why do we need to make this public?

Reply:
we don't ..good catch. switched it back.